ref/feature/openidConnectServer/description.adoc

```diff
@@ -0,0 +1,5 @@
+Single sign-on is a process to sign in once with your credentials to access multiple applications.
```

Comment:
For consistency purposes, I think you should reference the abbreviation for Single sign-on first here so change "Single sign-on to Single sign-on (SSO).